internal/pkg/service/stream/storage/node/coordinator/fileimport/operator.go

```diff
@@ -264,20 +265,21 @@ func (o *operator) checkFile(ctx context.Context, file *fileData) {
 }
 
 func (o *operator) importFile(ctx context.Context, file *fileData) {
-	o.logger.Info(ctx, "importing file")
-
 	ctx, cancel := context.WithTimeout(context.WithoutCancel(ctx), o.config.FileImportTimeout.Duration())
 	defer cancel()
 
+	o.logger.Info(ctx, "importing file")
+
 	// Lock all file operations
-	lock, unlock := clusterlock.LockFile(ctx, o.locks, o.logger, file.FileKey)
-	if unlock == nil {
+	lock, unlock, err := clusterlock.LockFile(ctx, o.locks, o.logger, file.FileKey)
+	if err != nil {
+		o.logger.Errorf(ctx, `file import lock error: %s`, err)
 		return
```

Comment:
I went through all the operators, since they are already merged, and unified the code to make it as similar as possible.

I expected you to come to an agreement, but it was not the same in the `main`.

Of course, it can be written differently, but when it is different in each operator, it is difficult to find errors.